sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1759,6 +1814,8 @@ def serialize(side_inputs):
         'triggering_frequency': self.triggering_frequency,
         'validate': self._validate,
         'temp_file_format': self._temp_file_format,
+        'ignore_insert_ids': self._ignore_insert_ids,
```

Comment:
`ignore_insert_ids` was previously not added to runner api parameter. Wondering if this is a bug as when we convert from the runner api the parameter will be lost.

Reply:
ah thanks Siyuan!